qt/python/mantidqt/widgets/codeeditor/interpreter.py

```diff
@@ -39,7 +41,10 @@ def __init__(self, editor):
         self.editor = editor
 
     def ask_for_filename(self):
-        filename, _ = QFileDialog.getSaveFileName(self.editor, "Choose filename...")
+        filename = open_a_file_dialog(self.editor, ".py", None, "Python Files (*.py)", True, True)
```

Comment:
Use named parameters here. That way it is obvious what parameters mean without having to look at the source for the function.